From 724efaa6ddffddc839ae3413a07d2424624cc8e5 Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Thu, 20 Jun 2019 20:39:05 +0300
Subject: [PATCH] ARM: dts: tegra20-glide: Fixup MMC bindings

This is needed to avoid some warnings at boot. Someone reported that
card detection is broken, but these changes should fix this too.

Also WiFi timings are bumped to avoid issues on suspend-resume.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 arch/arm/boot/dts/tegra20-glide.dts | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/tegra20-glide.dts b/arch/arm/boot/dts/tegra20-glide.dts
index 56e5407b2f42..ef5145ce16ac 100644
--- a/arch/arm/boot/dts/tegra20-glide.dts
+++ b/arch/arm/boot/dts/tegra20-glide.dts
@@ -1559,8 +1559,8 @@
 		compatible = "mmc-pwrseq-simple";
 		clocks = <&tegra_car TEGRA20_CLK_BLINK>;
 		clock-names = "ext_clock";
-		power-off-delay-us = <100000>;
-		post-power-on-delay-ms = <100>;
+		power-off-delay-us = <200000>;
+		post-power-on-delay-ms = <200>;
 	};
 
 	sdhci@c8000000 {
@@ -1571,6 +1571,7 @@
 		non-removable;
 		wakeup-source;
 		post-power-on-delay-ms = <100>;
+		no-1-8-v;
 
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -1590,8 +1591,11 @@
 		// cd-gpios = <&stmpegpio 16 GPIO_ACTIVE_HIGH>;
 		cd-gpios = <&gpio TEGRA_GPIO(V, 6) GPIO_ACTIVE_HIGH>;
 		vmmc-supply = <&vmmc_reg>;
+		post-power-on-delay-ms = <100>;
+		disable-wp;
 		broken-cd;
 		keep-power-in-suspend;
+		no-1-8-v;
 	};
 
 	// internal memory
@@ -1599,6 +1603,7 @@
 		status = "okay";
 		bus-width = <8>;
 		non-removable;
+		disable-wp;
 		keep-power-in-suspend;
 	};
 
-- 
2.22.0

